samples/react-native/navigation/index.windows.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ * @flow
+ */
+
+import React, { Component } from 'react';
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
+
```

Comment:
^^ Above imports should no longer be necessary, elmish-react imports all the necessary react stuff now.